apps/src/aichat/views/AichatView.tsx

```diff
@@ -41,34 +51,80 @@ const AichatView: React.FunctionComponent = () => {
     dispatch(setAiCustomizations(aiCustomizations));
   }, [dispatch, initialAiCustomizations]);
 
+  const aiCustomizations = useAppSelector(
+    state => state.aichat.currentAiCustomizations
+  );
+
+  const viewModeButtonsProps: SegmentedButtonsProps = {
+    buttons: [
+      {
+        label: 'Edit',
+        value: ViewMode.EDIT,
+        iconLeft: {iconName: 'wrench', iconStyle: 'solid', title: 'check'},
```

Comment:
Is "check" the right title here? Reading about semantic icons here:

https://docs.fontawesome.com/web/dig-deeper/accessibility#icons-used-as-semantic-elements

Reply:
And thanks for the reference - very useful!